cmake/UseMiniupnpc.cmake

```diff
@@ -1,7 +1,7 @@
 function(eth_apply TARGET REQUIRED)	
 	find_package (Miniupnpc 1.8.2013)
 	eth_show_dependency(MINIUPNPC miniupnpc)
-	if (MINIUPNPC_FOUND)
+	if (NOT NO_MINIUPNPC AND MINIUPNPC_FOUND)
```

Comment:
can you also add it to [EthOptions](https://github.com/ethereum/webthree-helpers/blob/develop/cmake/EthOptions.cmake) ?